src/modules/categories/category.routes.ts

```diff
@@ -71,10 +70,6 @@ export const createCategoryRoute = createRoute({
       }),
       "Category already exists",
     ),
-    [HTTPStatusCodes.UNPROCESSABLE_ENTITY]: jsonContent(
```

Comment:
This has to be there so that the zod errors will be shown automatically.

Reply:
Ya. I think i mistakenly removed it. Revert now.